prac_06/programming_language.py

```diff
@@ -0,0 +1,19 @@
+"""Prac06 ProgrammingLanguage Class"""
+
+
+class ProgrammingLanguage:
+
+    def __init__(self, name, typing, reflection, year):
+        self.name = name
+        self.typing = typing
+        self.reflection = reflection
+        self.year = year
+
+    def __str__(self):
+        return f"{self.name}, {self.typing} Typing, Reflection={self.reflection}, First appeared in {self.year}"
+
+    def is_dynamic(self):
+        if self.typing.lower() == "dynamic":
```

Comment:
Do we need an `if` statement here? Will `return self.typing.lower() == dynamic` have the same effect?

Reply:
Noted and fixed.